### DISCUSSION TITLE
TeensyStep <algorithm> lib issue

### DISCUSSION THREAD

**Discussion by AH-TNT** [luni64/TeensyStep#159] [General]
Hello Team,
I'm using TeensyStep lib to controll a step motor (just using a basic exemple), controlled by a Arduino Mega2560. Using VS Code, when I did compile I get a error on Stepper.h about a missing <algorithm> lib:
In file included from .pio\libdeps\megaatmega2560\TeensyStep\src\Stepper.cpp:1:0:
.pio\libdeps\megaatmega2560\TeensyStep\src\Stepper.h:3:10: fatal error: algorithm: No such file or directory
 #include "algorithm"
        
I tryed with Arduino IDE 2.2.1 and I get the same error... 
Could you pls help to fix this issue?
Thank you in advance,
AH

**Comment by luni64** [comment]
This library is only compatible to Teensy 3.x processors

**Reply by AH-TNT** [reply]
Hi luni64,
Thx to your clear msg.
I don't understand why VSCode (PlatformIO) inform the compability of your lib with platforms: Atmel AVR, Atmel Mega, Espressif 32, Espressif 8266, Raspberry Pico (RP2040), etc.....?!
Kregards,
AH

**Reply by luni64** [reply]
Traditionally the manifest for Teensy compatible libraries contains compatibility to AVR (which is clearly wrong, but is still used). Apparently PIO uses this information for showing the compatibility.

**Reply by AH-TNT** [reply]
Thank you to your explaination!
Regards,
AH
